test/apps/iodemo/io_demo.cc

```diff
@@ -2417,6 +2403,19 @@ class DemoClient : public P2pDemoCommon {
                         std::max(num_completed, io_op_perf_info[op].max);
                 io_op_perf_info[op].total       += num_completed;
                 io_op_perf_info[op].total_bytes += bytes_completed;
+
+                if (opts().per_conn_info) {
+                    double mbs = (bytes_completed / (elapsed * UCS_MBYTE));
+                    long iops  = (long)(num_completed / elapsed);
+
+                    if (op != IO_OP_MAX) {
```

Comment:
can we remove some duplication? only the name and last "|" are different

Reply:
done